### DISCUSSION TITLE
Having an issue with a missing component box in the inspector.

### DISCUSSION THREAD

**Discussion by Dayymer** [TokisanGames/Terrain3D#401] [Questions] [answered]
I don't see a way to just delete this, in typical fashion for me I immediately discovered the solution to my problem once I posted about it.

For any others that are having a problem where the node3d lets you create the new Terrain3d, but you don't get a textures box, go back in and turn the add on, for some reason it recognizes its there and lets you create the nodes, but isn't actually on, so I wasn't getting the box.

**Comment by TokisanGames** [comment, marked as answer]
Enabling the plugin in the project settings is listed in the instructions.
